src/jsifier.js

```diff
@@ -114,6 +114,19 @@ function JSify(data, functionsOnly) {
 
   // functionStub
   function functionStubHandler(item) {
+    // In LLVM, exceptions generate a set of functions of form __cxa_find_matching_catch_1(), __cxa_find_matching_catch_2(), etc.
+    // where the number specifies the number of arguments. In Emscripten, route all these to a single function '__cxa_find_matching_catch'
+    // that variadically processes all of these functions using JS 'arguments' object.
+    if (item.ident.startsWith('___cxa_find_matching_catch_')) {
+      if (DISABLE_EXCEPTION_THROWING) {
+        return error('DISABLE_EXCEPTION_THROWING was set (likely due to -fno-exceptions), which means no C++ exception throwing support code is linked in, but exception catching code appears. Either do not set DISABLE_EXCEPTION_THROWING (if you do want exception throwing) or compile all source files with -fno-except (so that no exceptions support code is required); also make sure DISABLE_EXCEPTION_CATCHING is set to the right value - if you want exceptions, it should be off, and vice versa.');
```

Comment:
Doesn't error raise an exception?  in which case the return is not needed, right?

Reply:
Oh, no, it does not. It logs and error, and tells the program to exit with an error. But it continues. That's how it can print an error on more than one missing symbol.

Reply:
Oh, I see.   Well at least maybe add the `return` on its own line.  Right now it looks like the returning the error is somehow what we are doing.

Reply:
I can rename `error` to something else if you have a better idea? (separate from this PR)

Reply:
Sure, moving the return.